api/leaderworkerset/v1/leaderworkerset_types.go

```diff
@@ -180,6 +188,27 @@ type SubGroupPolicy struct {
 	SubGroupSize *int32 `json:"subGroupSize,omitempty"`
 }
 
+type NetworkConfig struct {
+	// SubdomainPolicy determines the policy that will be used when creating
+	// the headless service
+	SubdomainPolicy SubdomainPolicy `json:"subdomainPolicy"`
```

Comment:
should be a pointer as well and mention that the default is Shared

Reply:
Changed